### ISSUE TITLE
Make font style configurable

### ISSUE THREAD

**Issue by very-doge-wow** [LunaticMuch/docusaurus-terminology#12]
Currently, the property font-weight is hardcoded:
https://github.com/LunaticMuch/docusaurus-terminology/blob/2a53632fddb412a02cd2f31067d5085866183664/components/tooltip.js#L9

I would like to be able to change this using custom CSS.

**Comment by LunaticMuch** [comment]
Good point...I am short of time this, might do in the week. Or I am happy if you submit a PR.
